tests/sentry/tasks/integrations/github/test_pr_comment.py

```diff
@@ -178,3 +179,21 @@ def test_multiple_orgs(self):
             self.another_organization.id,
             [groupowner_2.id],
         )
+
+    def test_format_comment(self):
```

Comment:
Needs one more test to verify the entire comment body looks correct

Reply:
i'll just modify this test to check the entire comment body